### PR TITLE
Content/add project card for bagel

### PR DESCRIPTION
**Pull request by JacOng17**
Added content & project card for Bagel.
image

### CODE REVIEW

File: _services_projects/youth-voice.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+agency: Ministry for Youth Development
+title: Youth Voice
+subtitle:
+permalink: /projects/youth-voice/
+redirect_from: /projects/youth-voice/
+excerpt: Exploring how machine learning can help assist policy analysts in their decision making.
```
Comment (mermop) [inline _services_projects/youth-voice.md]: "help assist" is redundant, and we can make this sentence more active

```suggestion
excerpt: Exploring how machine learning can help policy analysts make sense of public feedback.
```

File: _services_projects/youth-voice.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    - "[bagel-jam](https://github.com/ServiceInnovationLab/bagel-jam)"
+report:
+media:
+    - "[Smart Policy Analysis Tools](https://serviceinnovationlab.github.io/bagel-box/)"
+    - "[The Hive](https://www.the-hive.nz/)"
+---
+
+Exploring how machine learning can help assist policy analysts in their decision making.
+
+MYD have been working on a new online submission tool for youth called [The Hive](https://www.the-hive.nz/). The Hive has proved to be a good tool to potentially receive large amounts of submissions from youth. MYDs policy team were concerned about their ability to process large amounts of submission data. There was an opportunity to explore how data is received and analysed and how the analysis process could be improved through the use of data analysis tools.
```
Comment (mermop) [inline _services_projects/youth-voice.md]:
```suggestion
The Ministry of Youth Development have been working on a new online submission tool for youth called [The Hive](https://www.the-hive.nz/). The Hive has proven to be a great tool for young people to give feedback on issues across government, but policy teams aren't always equipped to process large amounts of submission data. There was an opportunity to explore how data is received and analysed and how the analysis process could be improved through the use of data analysis tools.
```

**Comment by mermop** [comment]
Do we have any photos from this project? @rosspatel01, have you got anything appropriate ?

**Comment by Br3nda** [comment]
Looks like it needs an image for the card

image

**Pull request merged**